locales/ko.json

```diff
@@ -0,0 +1,63 @@
+{
+  "ko": {
+    "translation": {
+      "__comment__": "Korean localization",
+      "texts": {
+        "will open soon": ", 하지만 곧 열릴",
+        "will close soon": ", 그러나 곧 문을 닫습니다",
```

Comment:
I missed this. What is the context around here? How to find this message on veggikarte?

Reply:
Found an example.

![Screenshot_20230621-225125](https://github.com/piratenpanda/veggiekarte/assets/28209361/a71501b8-b009-4a96-b389-f30991e18584)

Reply:
Here is an example:
![Screenshot from 2023-06-21 15-52-52](https://github.com/piratenpanda/veggiekarte/assets/35647502/7cf09ca6-164a-48bc-b2b7-575446f23143)